.gitignore

```diff
@@ -31,9 +31,6 @@ OPATH
 ehthumbs.db
 Thumbs.db
 
-# The binary
-trickster
```

Comment:
Do we want to ignore it in OPATH as well?

Reply:
OPATH is already in .gitignore so we're good